mlir/lib/Dialect/SPIRV/Transforms/UpdateVCEPass.cpp

```diff
@@ -95,6 +95,16 @@ static LogicalResult checkAndUpdateCapabilityRequirements(
   return success();
 }
 
+static SetVector<spirv::Capability>
+withImpliedCapabilities(SetVector<spirv::Capability> &caps) {
+  SetVector<spirv::Capability> allCaps(caps.begin(), caps.end());
+  for (auto cap : caps) {
```

Comment:
```suggestion
  for (spirv::Capability cap : caps) {
```
See https://llvm.org/docs/CodingStandards.html#use-auto-type-deduction-to-make-code-more-readable